ZukoAzula2016/src/org/team1540/zukoazula/AutonomousBase.java

```diff
@@ -112,6 +114,12 @@ protected void eject(float time, float driveBackSpeed) throws AutonomousModeOver
         Shooter.stopEvent();
     }
 
+    protected void ejectWhileDrivingBackDistance(float feet, float speed) throws AutonomousModeOverException, InterruptedException {
```

Comment:
The reason I made it like this was because Zuko can't eject unless the robot is driving backwards. Otherwise it just doesn't work. I can probably clear that up in comments.